tasks/managed/apply-mapping/tests/test-apply-mapping-content-gateway-disabled.yaml

```diff
@@ -54,40 +68,98 @@ spec:
                 ]
               }
               EOF
+          - name: skip-trusted-artifact-operations
```

Comment:
Why are these steps here? They look the same as the one added in the task

Reply:
seeing as there is no workspace to share data between the `task under test` and the `check-results` task...we need to share via Trusted Artifacts.